lib/trento.pm

```diff
@@ -63,12 +68,96 @@ use constant TRENTO_AZ_PREFIX => 'openqa-trento';
 # Azure does not support dash or underscore in ACR name
 use constant TRENTO_AZ_ACR_PREFIX => 'openqatrentoacr';
 use constant CYPRESS_IMAGE_TAG => 'goofy';
+use constant CYPRESS_IMAGE => 'docker.io/cypress/included';
 
 =head1 DESCRIPTION 
 
-Package with common methods and default or constant  values for Trento tests
+Package with common methods and default or
+constant values for Trento tests
 
 =head2 Methods
+=cut
+
+=head3 get_trento_deployment
+
+Get the set of scripts for the Trento deployment
+=cut
+
+sub get_trento_deployment {
+    my ($self, $work_dir) = @_;
+
+    enter_cmd "cd $work_dir";
+    my $gitlab_token = get_var(TRENTO_GITLAB_TOKEN => get_required_var('_SECRET_TRENTO_GITLAB_TOKEN'));
+    script_run 'read -s GITLAB_TOKEN', 0;
+    type_password get_var(TRENTO_GITLAB_TOKEN => get_required_var('_SECRET_TRENTO_GITLAB_TOKEN')) . "\n";
+
+    # Script from a release
+    if (get_var('TRENTO_DEPLOY_VER')) {
+
+        # Clean up whatever is present in the indicated working folder
+        # of the qcow2 image, to avoid confusion
+        enter_cmd 'rm -rf *';
+
+        # Get some 'coordinates' from the test settings
+        my $ver = get_var('TRENTO_DEPLOY_VER');
+        my $gitlab_repo = get_var(TRENTO_GITLAB_REPO => 'gitlab.suse.de/qa-css/trento');
+        my @gitlab_url = split('/', $gitlab_repo);
+        my $gitlab_namespace = $gitlab_url[-2];
+        my $gitlab_project = $gitlab_url[-1];
+
+        assert_script_run('echo "PRIVATE-TOKEN: ${GITLAB_TOKEN}" > gitlab_conf');
+        my $curl_cmd = 'curl -s ' .
+          '-H @gitlab_conf';
```

Comment:
There is not really a reason to have here a concatenation operation between two strings if both are constants, right? You could directly do `my $curl_cmd = 'curl -s -H @gitlab_conf';`

Reply:
I like to split them to facilitate the code reader to understand different sections of the command